Add the ability to run syft from a scratch image. (#288)
anchore/syft: README.md

brew install syft
```

> **Container Image**

Reviewer: ```suggestion
**Docker**
```

Reviewer 2: Why do we want to say Docker instead of container image?

Author: I think container image is a better term as this will allow us to include other engines in the future.

Reviewer: actually, on further consideration, I retract this...

> Example of how to use Syft scratch image:

Reviewer: ```suggestion
Example of how to use Syft with docker:
```

Reviewer: (in this way we just use the bolded title/section to hint at the installation mode)

```bash

> docker pull anchore/syft:latest

Reviewer: ```suggestion

```
technically docker will attempt to pull the image, so saves example space

> docker run -v /var/run/docker.sock:/var/run/docker.sock anchore/syft -o json alpine:latest

Reviewer: ```suggestion
docker run -v /var/run/docker.sock:/var/run/docker.sock anchore/syft:latest -o json alpine:latest
```

Reviewer: also, should we throw in `-t` to get the ETUI output? or leave as is for logger output? (open question)

Author: So I added the "-t" or Allocate a pseudo TTY option and the terminal hung, which is odd.

Reviewer: gotcha, we'll leave this as is 👍  thanks for confirming
